meta-lmp-bsp/recipes-security/optee/optee-os-fio-bsp-common.inc

```diff
@@ -3,6 +3,28 @@ OPTEE_HEADER:stm32mp1common    = "tee-header_v2"
 OPTEE_PAGEABLE:stm32mp1common  = "tee-pageable_v2"
 OPTEE_PAGER:stm32mp1common     = "tee-pager_v2"
 
+# TI HS signing logic (from meta-ti-bsp optee-os bbappend)
+do_compile:prepend:k3() {
+    export TI_SECURE_DEV_PKG=${TI_SECURE_DEV_PKG}
+}
+# Signing procedure for K3 HS devices
+optee_sign_k3hs() {
+    ( cd ${B}/core/; \
+        if [ -f ${TI_SECURE_DEV_PKG}/scripts/secure-binary-image.sh ]; then \
+            ${TI_SECURE_DEV_PKG}/scripts/secure-binary-image.sh tee-pager_v2.bin tee-pager_v2.bin.signed; \
+        else \
+            echo "Warning: TI_SECURE_DEV_PKG not set, OP-TEE not signed."; \
+            cp tee-pager_v2.bin tee-pager_v2.bin.signed; \
```

Comment:
So I think because TI_SECURE_DEV_PKG is not defined it will always fall here.
Need to check locally

Reply:
Yeah, this is expected, that way we have only one binary that work for all cases (signed and not signed).